### DISCUSSION TITLE
c++ compiler flags

### DISCUSSION THREAD

**Discussion by k34n4y138** [geeksblabla/blanat#47] [Q&A] [answered]
some STD libraries need to be linked manually during compilation.
how is that managed?

**Comment by k34n4y138** [comment]
the threading library requires -lpthread flag present in the compilation command

**Reply by aboullaite** [reply]
We do not support passing flags for now.

**Reply by dauom** [reply, marked as answer]
you can use the standard threading library `std::thread` (https://en.cppreference.com/w/cpp/thread/thread) instead of pthread.
